Users/Tutorial.rst

```diff
@@ -66,12 +66,57 @@ coala will now ask you for missing values that are needed to perform the
 analysis, which in this case is only the ``use_spaces`` setting. We
 recommend setting it to ``True``.
 
+::
+
+    Please enter a value for the setting "use_spaces" (True if spaces
+    are to be used instead of tabs.) needed by SpaceConsistencyBear
+    for section "cli"
+
+We recomment setting it to ``True``.
```

Comment:
--> recommend
However, this is duplicated from above, please remove this sentence.